PiwikTracker-Swift/Dictionary+Extension.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Dictionary+Extension.swift
+//  PiwikTracker
+//
+//  Created by Cornelius Horstmann on 11.10.16.
+//  Copyright © 2016 Mattias Levin. All rights reserved.
+//
+
+import Foundation
+
+protocol OptionalType {}
+
+extension Optional: OptionalType {}
+
+extension Dictionary {
+    static func removeOptionals(dictionary: [Key: Value?]) -> [Key:Value] {
```

Comment:
Since this is a static method I would probably not put in an extension.
The alternative would be to make this a generic instance method.